src/styles/components/accordion.scss

```diff
@@ -0,0 +1,50 @@
+@import '../colours';
+
+.accordion {
+  width: 100%;
+  border: 1px solid $medium-gray;
```

Comment:
Not sure where this colour is defined (probably Foundation) but we use the ones in `./colours.scss` - in this case it would be `$colour-light-grey;`

Reply:
Updated